imageio/core/request.py

```diff
@@ -112,6 +112,15 @@ class Mode(str, enum.Enum):
     write_multi_volume = "wV"
     write_any = "w?"
 
+    @classmethod
+    def _missing_(cls, value):
```

Comment:
What is this for? Maybe a docstring explaining its purpose?

Reply:
It's a standard thing (that I didn't know about before stumbling upon it neither): https://docs.python.org/3/library/enum.html#supported-sunder-names

We use it to allow a `Mode` to be created from "r" and "w" respectively, i.e., it maps `r->r?` and `w->w?`. The new API only cares about the `IOMode` of the request object, but the `ImageMode` is still needed for legacy reasons.

What was your confusion about? the method name, or it's purpose?

Reply:
> What was your confusion about? the method name, or it's purpose?

Both. If we can add what you wrote here as a docstring we'd avoid confusion for future readers :)

Reply:
Absolutely. I mainly asked to know what to write into the docstring :)

Reply:
Done. @almarklein Could you check the docstring that I added and make sure it makes sense to you, too?

Reply:
Very clear! Thanks!